puppet/hiera/common.yaml

```diff
@@ -36,8 +58,8 @@ uber::db::user:    '%{hiera("uber::config::db_user")}'
 uber::db::pass:    '%{hiera("uber::config::db_pass")}'
 uber::db::dbname:    '%{hiera("uber::config::db_name")}'
 
-uber::config::code_of_conduct: "http://coolcon.com/codeofconduct"
-uber::config::contact_url: "contact@coolcon.org"
+uber::config::code_of_conduct: "http://www.imdb.com/title/tt0096928/quotes?item=qt0412295"
+uber::config::contact_url: "noreply@coolcon.org"
```

Comment:
+1 on the new Code of Conduct link!

I believe that the contact URL is an actual URL and not an email address BTW.  So this is gonna make for a weird and broken link in the test system.  Maybe change it to http://lmgtfy.com/ ?